corehq/apps/saved_reports/management/commands/fix_saved_reports_filters.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from dimagi.utils.couch.cache import cache_core
+
+from corehq.apps.saved_reports.models import ReportConfig
+from corehq.apps.userreports.reports.filters.values import CHOICE_DELIMITER
+
+
+class Command(BaseCommand):
+    """
+        Migrated ReportConfig filters with multiple values from CHOICE_DELIMITER-delimited strings
+        to lists. This command can be deleted once that migration is complete.
+    """
+    def handle(self, *args, **options):
+        key = ["name slug"]
+        results = cache_core.cached_view(
```

Comment:
Curious why you're using the cached view and not going straight to couch?

Reply:
dbb5fa96944

Reply:
dbb5fa96944